site/docs/v1/tech/admin-guide/upgrade.adoc

```diff
@@ -7,41 +7,270 @@ navcategory: admin
+* Monitoring the https://github.com/FusionAuth/[FusionAuth GitHub repository].
+
+== How to Think About Upgrades
+
+Before upgrading FusionAuth, there are a number of things to consider. Go through each of the points below to see if any of them apply to your situation.
+
+=== Update Contents and Notes
+
+Read the link:/docs/v1/tech/release-notes[release notes]. Release notes help you understand how an update affects you. They include details on new features, improvements, bug fixes, and potential breaking changes.
+
+=== Testing
+
+Before starting the upgrade process on production, test the upgrade in a staging or development environment to identify potential issues. This is particularly important on more complex upgrades involving multiple version jumps or if you have made customizations to FusionAuth.
+
+=== Downtime and Database Migrations
+
+Plan for any downtime or maintenance windows required for the upgrade. If changes to the database schema are required, this could require more steps and downtime that affects end users. A pure application upgrade, on the other hand, may only affect admin functionality and not require any downtime from an end-user perspective.
+
+FusionAuth can perform database migrations automatically and silently using a process called link:/docs/v1/tech/guides/silent-mode[Silent Mode]. 
+
+If you want to run the migrations interactively, use link:/docs/v1/tech/installation-guide/fusionauth-app#maintenance-mode[Maintenance Mode], or you can run any needed scripts out of band (meaning directly on the database yourself).
+
+[NOTE]
+====
+You should always back up your database before using Maintenance Mode or performing any database migrations by any method.
+====
+
+The timing of the database upgrade is also important to note. When an upgraded node is added to a FusionAuth cluster, it will lock the database and perform the database upgrade if configured for Silent Mode. This means that the database will be unavailable to other nodes in the cluster until the upgrade is complete. 
+
+=== Connected Applications and Client Libraries
+
+Consider that a FusionAuth server upgrade might mean that you need to upgrade your client libraries too. Check if this is needed before upgrading, as it will entail a more complex upgrade process and may require more downtime.
+
+=== Skipping Versions 
+
+Sometimes, you will want to upgrade to a FusionAuth version that is a few versions newer than your current version. This is called a "version jump". FusionAuth migration can run all the necessary database migrations to get you to the latest version. However, we recommend that you don't skip minor versions and rather migrate from one minor version to the next, step by step. For example, if you are on version 1.37 and want to get to version 1.43, we recommend that you upgrade each minor step to 1.43.
```

Comment:
Can you please use real version numbers?

Also, I'd counsel folks to review the release notes. If the releases focus on SAML or another feature that isn't used, version jumps are okay.

Also, test in staging with real data. Test both for functionality changes and for the time it takes to update your real data.

Reply:
Is this resolved @sixhobbits ?